src/PlexShareScreenshare/Client/Screenshot.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Drawing.Imaging;
+using SharpDX;
+using SharpDX.Direct3D11;
+using SharpDX.DXGI;
+
+using Device = SharpDX.Direct3D11.Device;
+using MapFlags = SharpDX.Direct3D11.MapFlags;
+using Resource = SharpDX.DXGI.Resource;
+
+namespace PlexShareScreenshare.Client
+{
+    internal class Screenshot
+    {
```

Comment:
Do we need to add unit tests in this class, since this class essentially has code from a library?

Reply:
To be updated later